mmv1/third_party/terraform/services/resourcemanager/ephemeral_google_service_account_key.go

```diff
@@ -38,32 +38,39 @@ type ephemeralServiceAccountKeyModel struct {
 
 func (p *googleEphemeralServiceAccountKey) Schema(ctx context.Context, req ephemeral.SchemaRequest, resp *ephemeral.SchemaResponse) {
 	resp.Schema = schema.Schema{
+		Description: "Get an ephemeral service account public key.",
 		Attributes: map[string]schema.Attribute{
 			"name": schema.StringAttribute{
-				Required: true,
+				Description: "The name of the service account key. This must have format `projects/{PROJECT_ID}/serviceAccounts/{ACCOUNT}/keys/{KEYID}`, where `{ACCOUNT}` is the email address or unique id of the service account.",
+				Required:    true,
 				Validators: []validator.String{
 					stringvalidator.RegexMatches(
 						regexp.MustCompile(verify.ServiceAccountKeyNameRegex),
 						"must match regex: "+verify.ServiceAccountKeyNameRegex,
 					),
 				}},
 			"public_key_type": schema.StringAttribute{
-				Optional: true,
+				Description: "The output format of the public key requested. TYPE_X509_PEM_FILE is the default output format.",
+				Optional:    true,
 				Validators: []validator.String{
 					stringvalidator.OneOf(
+						"TYPE_NONE",
```

Comment:
Should TYPE_NONE be included here? I think in the API this value represents the field not being set by the user, so the API will provide a default value of TYPE_X509_PEM_FILE in that case. 

I see the default is implemented in the Open method so all good there 👍🏻

Reply:
just removed, doesn't make sense to include. The same is applied to all enums also that have a NONE value.